scripts/component_emphasize/component_emphasize.gml

```diff
@@ -0,0 +1,25 @@
+// Script assets have changed for v2.3.0 see
+// https://help.yoyogames.com/hc/en-us/articles/360005277377 for more information
+function component_emphasize(str, substr = "", tag = -1) {
+	
+	if (substr == "") {
+		substr = str;
+	}
+	
+	var last_pos = 0;
+	while (true) {
+		var pos = string_pos_ext(substr,str,last_pos);
+		if (pos == 0) break;
+		last_pos = pos + string_length(substr) + 4;
+		str = string_insert("}}",str,pos + string_length(substr));
+		if (tag != -1) {
+			str = string_insert("|",str,pos);
+			str = string_insert(string(tag),str,pos);
+			last_pos += string_length(string(tag)) + 1;
+		}
+		str = string_insert("{{",str,pos);
+	}
+	
+	return str;
+	
+}
```

Comment:
I think with the simpler direction, this may not be needed.

Reply:
It's never needed. This is an optional function that can be used to programmatically emphasize a word or phrase. If you never need it, you won't run it and it won't slow anything down. If you're sure you will absolutely never need to programmatically emphasize something, the program can be removed without any negative consequences.

Reply:
I think `draw_container` should be the only way we interface with this system.  Therefore, I think we can remove this.

Reply:
This doesn't actually interface with this system - all it does is put asterisks around a string. It's a preprocessing step.

Reply:
Yes, let's remove this.